db/migrate/20160921184937_add_cancel_reason_detail_to_replication_transfer.rb

```diff
@@ -0,0 +1,5 @@
+class AddCancelReasonDetailToReplicationTransfer < ActiveRecord::Migration
+  def change
+    add_column :replication_transfers, :cancel_reason_detail, :text, null: true
+  end
+end
```

Comment:
We should consider the overhead of syncing a large reason field. Are we trying to encapsulate the entire failure stack or a pointer to local information?